src/main/java/net/snowflake/ingest/streaming/internal/StreamingIngestUtils.java

```diff
@@ -122,4 +122,12 @@ public static String getShortname(final String fullname) {
     final String[] parts = fullname.split("/");
     return parts[parts.length - 1];
   }
+
+  public static String getPayloadAsStr(Map<Object, Object> payload, String message) {
```

Comment:
nit: javadocs on public method.

Reply:
method removed, so will not add javadocs